### ISSUE TITLE
Bug Report: Leaked instance: LuaAPI:id

### ISSUE THREAD

**Issue by Trey2k** [WeaselGames/godot_luaAPI#66]
**Describe the bug**
Memory leak cause when ever a new instance and the LuaAPI is created. Get the following warning in stderr `Leaked instance: LuaAPI:id`

**To Reproduce**
call luaAPI.new() and store it in a variable
```gdscript
extends Node2D

func _ready():
	var lua = LuaAPI.new()
	var err = lua.do_string("print('test')")
	if err is LuaError:
		print("ERROR %d: " % err.type + err.msg)
	get_tree().quit(0)
```

**Expected behavior**
To not leak.

**Enviromint (please complete the following information):**
 - OS: Windows 10 and Linux ubuntu 20
 - Godot version: commit b2e3758
 - Module version: [ed22b27](https://github.com/WeaselGames/godot_luaAPI/pull/65/commits/ed22b278505e2152bad16903cce7150b4358c470)

**Additional context**
It has to do with `ObjectDB::remove_instance` not being called. Im not familiar enough with this part of godot yet to say why that is.

**Comment by Trey2k** [comment]
I logged all object IDs passed to `ObjectDB::add_instance`, and the ID listend in the Leaked instance was not there. So I am unsure how else it could be added to `object_slots`.

**Comment by Trey2k** [comment]
Fixed in a5b895150000ff8f46522c2af651c3e08c02efe2 will merge with #65
